src/ontology/cl-edit.owl

```diff
@@ -29470,6 +29471,20 @@ SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000005444))
 SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000015665))
 
+# Class: obo:CL_4023122 (oxytocin receptor sst GABAergic cortical interneuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25303526"^^xsd:string) obo:IAO_0000115 obo:CL_4023122 "An interneuron located in the mouse cerebral cortex that express the oxytocin receptor. These interneurons also express somatostatin.")
```

Comment:
@shawntanzk if you agree to make the term broadly applicable beyond mouse, I'd remove "mouse" from the definition ;-)
You may add an rdfs: comment or a definition gloss stating that the cell type was first identified in mouse if you wish.

Reply:
whoops >.< thanks for the catch! and yes, a comment is a good idea :) thanks